src/App.tsx

```diff
@@ -1,109 +1,26 @@
 import React from 'react';
-
 import './App.scss';
+import postsFromServer from './api/posts';
+import commentsFromServer from './api/comments';
+import usersFromServer from './api/users';
+import { Post } from './types/Post';
+import { PostList } from './components/PostList';
+
+type InitialPosts = Omit<Post, 'user' | 'comments'>;
+
+function getInformativePosts(initialPosts: InitialPosts[]): Post[] {
+  return initialPosts.map(post => ({
+    ...post,
+    user: usersFromServer.find(userFind => userFind.id === post.userId) || null,
+    comments: commentsFromServer.filter(comment => comment.postId === post.id),
+  }));
+}
 
-// import postsFromServer from './api/posts';
-// import commentsFromServer from './api/comments';
-// import usersFromServer from './api/users';
+const posts: Post[] = getInformativePosts(postsFromServer);
```

Comment:
unnecessary, bc you already give type to your function
```suggestion
const posts = getInformativePosts(postsFromServer);
```